packages/subgraph/tests/utils/events/osx.ts

```diff
@@ -1,9 +1,7 @@
-// TODO: Remove this file and import from OSx-commons-subgraph,
-// once the OSx-commons-subgraph npm package is published
 import {
   InstallationPrepared,
   InstallationPreparedPreparedSetupDataStruct,
-} from '../generated/PluginSetupProcessor/PluginSetupProcessor';
+} from '../../../generated/PluginSetupProcessor/PluginSetupProcessor';
```

Comment:
When I see this level of nesting, it really does suggest setting up [paths](https://www.typescriptlang.org/tsconfig#paths)

Reply:
I created a task for it https://aragonassociation.atlassian.net/browse/OS-1047